docs/python-schema.md

```diff
@@ -0,0 +1,42 @@
+# Python Schema for Excel CPF Templates
```

Comment:
Only note here is maybe we can link the developers to what the excel template looks like? In ARPA we commit to code all the versions of the input templates, and I'm wondering if something like that would make sense for CPF.

Alternatively we can link the Google Drive that includes the input templates.

Reply:
Oooh yeah I think perhaps linking to the google drive would be nice / easiest to make sure they have access to different versions? I can add in a subsequent PR